deepmd/env.py

```diff
@@ -120,6 +121,29 @@
         r"model_attr/t_tab_data|"
 )
 
+REMOVE_SUFFIX_DICT = {
```

Comment:
`fitting_attr`?

Reply:
`fitting_attr_{}/` is contained in situation "_{}/" --> "/".